emu/default.cfg

```diff
@@ -13,6 +13,7 @@ allowedinputpaths = /
 maxsingleinputsize = 200mb
 maxprocesses = 10
 parallelprocesses = 2
+languages = en-US,fr-CA
```

Comment:
config parameter is `language` ... no `s`.

Reply:
made a local change. works with fixed parameter name.

Reply:
Ahhh ! Thanks !